rts/Makefile

```diff
@@ -136,9 +138,21 @@ _build/wasm32:
 # Common configuration
 # 
 
-RTS_EMSCRIPTEN_64=emcc -c --target=wasm64-unknown-emscripten -fpic
-RTS_EMSCRIPTEN_64_DEBUG=$(RTS_EMSCRIPTEN_64)
-RTS_EMSCRIPTEN_64_RELEASE=$(RTS_EMSCRIPTEN_64) -O2
+# Arguments obtained by Emscripten 3.1.64:
+# `emcc -c --target=wasm64-unknown-emscripten -fpic --verbose`
+# while skipping unneeded arguments.
+RTS_LL_64=$(CLANG) -cc1 -triple wasm64-unknown-emscripten -emit-obj \
+	-mrelocation-model pic -pic-level 1 -mframe-pointer=none \
+	-ffp-contract=on -fno-rounding-math -mconstructor-aliases \
+	-target-feature +mutable-globals -fvisibility=hidden \
+	-resource-dir $(WASM_CLANG_LIB)/lib/clang/19/ \
```

Comment:
This should be `clang/18`. Probably the option is not needed and can be removed...